dcm4ceph-core/src/main/java/org/open_ortho/dcm4ceph/util/FileUtils.java

```diff
@@ -169,20 +171,18 @@ public static Properties loadProperties(String conffile) {
     public static Properties loadProperties(File filename) {
         Log.info("Loading Properties file " + filename);
         Properties props = new Properties();
-
-        if (!filename.canRead()) {
-            Log.warn("Cannot read from file " + filename + ".");
-        } else {
-            try {
-                props.load(new FileInputStream(filename));
-            } catch (FileNotFoundException e) {
-                e.printStackTrace();
-            } catch (IOException e) {
-                e.printStackTrace();
-            }
+        try (InputStream is = Files.newInputStream(filename.toPath())) {
+            props.load(is);
+            return props;
+        } catch (IOException e) {
+            e.printStackTrace();
+            Log.warn("Cannot read from file " + filename.toPath() + ".\n" +
```

Comment:
Change to error and remove the line below about using the defaults properties. Default properties file should be the same file base name of the image name, but with .properties extension.

Reply:
Done, added also reference to ceph_defaults as another example of .properties file.
One file is as real example, other as sensible defaults.